debian/changelog

```diff
@@ -0,0 +1,5 @@
+ooniprobe-cli (3.0.0-rc.9-1) UNRELEASED; urgency=medium
```

Comment:
This is probably `-rc.10`

Reply:
Is there a way to generate this from the git tags? Is there some pattern for doing that in debian land?